samza-kv-rocksdb/src/main/scala/org/apache/samza/storage/kv/RocksDbKeyValueStore.scala

```diff
@@ -20,89 +20,70 @@
 package org.apache.samza.storage.kv
 
 import java.io.File
+
 import org.apache.samza.SamzaException
-import org.apache.samza.util.{ LexicographicComparator, Logging }
 import org.apache.samza.config.Config
-import org.rocksdb._
-import org.rocksdb.TtlDB
+import org.apache.samza.util.{LexicographicComparator, Logging}
+import org.rocksdb.{TtlDB, _}
 
 object RocksDbKeyValueStore extends Logging {
 
-  def openDB(dir: File, options: Options, storeConfig: Config, isLoggedStore: Boolean, storeName: String, metrics: KeyValueStoreMetrics): RocksDB = {
+  def openDB(dir: File, options: Options, storeConfig: Config, isLoggedStore: Boolean,
+             storeName: String, metrics: KeyValueStoreMetrics): RocksDB = {
     var ttl = 0L
     var useTTL = false
 
-    if (storeConfig.containsKey("rocksdb.ttl.ms"))
-    {
-      try
-      {
+    if (storeConfig.containsKey("rocksdb.ttl.ms")) {
+      try {
         ttl = storeConfig.getLong("rocksdb.ttl.ms")
 
-        // RocksDB accepts TTL in seconds, convert ms to seconds
-        if(ttl > 0) {
-          if (ttl < 1000)
-          {
-            warn("The ttl values requested for %s is %d, which is less than 1000 (minimum), using 1000 instead",
-              storeName,
-              ttl)
+        if (ttl > 0) {
+          if (ttl < 1000) {
+            warn("The ttl value requested for %s is %d which is less than 1000 (minimum). " +
+              "Using 1000 ms instead.", storeName, ttl)
             ttl = 1000
           }
-          ttl = ttl / 1000
-        }
-        else {
-          warn("Non-positive TTL for RocksDB implies infinite TTL for the data. More Info -https://github.com/facebook/rocksdb/wiki/Time-to-Live")
+          ttl = ttl / 1000 // RocksDB accepts TTL in seconds, convert ms to seconds
```

Comment:
No comment needed if it were switched to this:
ttlS = TimeUnit.MILLISECONDS.toSeconds(ttlMs)